src/py-opentimelineio/opentimelineio-bindings/otio_serializableObjects.cpp

```diff
@@ -443,11 +452,11 @@ Base class for an :class:`~Item` that contains other :class:`~Item`\s.
 Should be subclassed (for example by :class:`.Track` and :class:`.Stack`), not used directly.
 )docstring")
         .def(py::init([](std::string name,
-                         py::object children,
+                         optional<std::vector<Composable*>> children,
```

Comment:
Should the comment above, noting "Item", be revised to say "Composable"?

Reply:
Good idea. I will update that.